src/Application.js

```diff
@@ -39,7 +64,12 @@ export class Application {
       soundVolume: new Property(1),
       musicVolume: new Property(1),
       voVolume: new Property(1),
-      sfxVolume: new Property(1)
+      sfxVolume: new Property(1),
+      pointerSize: new Property(0.05),
+      controlSensitivity: new Property(0.5),
+      buttonSize: new Property(0.5),
+      removableLayers: new Property({}),
```

Comment:
I'm not exactly sure how `removableLayers` should work. I see two options:
- `removableLayers` is a number between 0 and 1. As the number increases more layers are removed, with 1 being the "least distracting" version of the game.
- Or... `removableLayers` is configured by the developer ("My game has these layers: [bg, trees]"), and the property reflects which layers should be disabled. Maybe an object:
```javascript
{
  "bg": false,
  "trees": true
}
```

This would give devs a lot more control but require us to handle that from a UI standpoint at the container level a little bit better.